src/services/administration/iam/role/RolePage.vue

```diff
@@ -1,36 +1,62 @@
 <template>
     <section class="role-page">
-        <p-page-title title="Role" />
+        <p-page-title title="Role"
+                      use-selected-count
+                      use-total-count
+                      :total-count="totalCount"
+                      :selected-count="selectedIndex.length"
+        />
         <p-horizontal-layout>
-            <template #container="{ height }" />
+            <template #container="{ height }">
+                <role-management-table :table-height="height"
+                                       @update:selected-index="handleUpdate"
+                                       @update:total-count="handleUpdate"
+                />
+            </template>
         </p-horizontal-layout>
     </section>
 </template>
 
 <script lang="ts">
-import { reactive, toRefs } from '@vue/composition-api';
+import {
+    defineComponent, reactive, toRefs,
+} from '@vue/composition-api';
 
 import {
-    PHorizontalLayout, PPageTitle,
+    PHorizontalLayout, PPageTitle, PToolboxTable, PIconTextButton, PSelectDropdown,
 } from '@spaceone/design-system';
+import RoleManagementTable from '@/services/administration/iam/role/modules/RoleManagementTable.vue';
 
-export default {
+export default defineComponent({
     name: 'RolePage',
     components: {
         PHorizontalLayout,
         PPageTitle,
+        PToolboxTable,
+        PIconTextButton: PIconTextButton as any,
+        PSelectDropdown,
+        RoleManagementTable,
     },
     setup() {
         const state = reactive({
-            // selected
+            totalCount: 0,
+            selectedIndex: [] as number[],
```

Comment:
레거시 페이지들에서 헷갈리는 변수명을 사용하고 있긴 하지만...
새로운 페이지들에서는 명확한 변수명을 사용했으면 합니다.
```suggestion
            selectedIndices: [] as number[],
```